fuel-core/src/args.rs

```diff
@@ -41,6 +41,10 @@ pub struct Opt {
     #[clap(name = "CHAIN_CONFIG", long = "chain", default_value = "local_testnet")]
     pub chain_config: String,
 
+    /// Allows RPC Endpoints to arbitrarily modify the VM. Used for local development only
+    #[clap(long = "enable_rpc_control")]
+    pub enable_rpc_control: bool,
```

Comment:
We don't refer to our endpoints as RPC, as we don't use any rpc frameworks like json-rpc or grpc. I think we should have a special feature flag just for manual block production, and we could make super features out of the smaller ones later if we find that they are commonly enabled together.

Also, we should use kebab casing like our other arg flags (`vm-backtrack`, `utxo-validation`).

```suggestion
    #[clap(long = "manual-blocks")]
    pub manual_blocks_enabled: bool,
```